src/encoded/static/components/filegallery.js

```diff
@@ -1195,7 +1195,9 @@ class FilterControls extends React.Component {
 
     render() {
         const { filterOptions, selectedFilterValue, browsers, currentBrowser, browserChangeHandler, visualizeHandler, context } = this.props;
-        const contextFiles = context.files || [];
+        const filterIncArchive = document.querySelector('[name="filterIncArchive"]');
+        const filterIncArchiveStatus = filterIncArchive ? filterIncArchive.checked : false;
```

Comment:
I feel like instead of querying the DOM, you should pass the FileGalleryRendererComponent `inclusionOn` state as a prop to `FilterControls`. Seems more “Reactic,” and also when `inclusionOn` changes state, `FilterControls` gets notified of this. Same thing for `handleDownloadClick`.

Reply:
Done